Makefile

```diff
@@ -10,13 +10,29 @@ else
 endif
 
 export VERSION?=$(shell (git describe --abbrev=0 --tags | sed -e 's/v//') || echo $(cat VERSION)-$(git log -1 --pretty='%h'))
-PRERELEASE=`grep -q dev <<< "${VERSION}" && echo "pre" || echo ""`
+PRERELEASE=`echo "${VERSION}" | grep -q dev && echo "pre" || echo ""`
 REVISION?=`git log -1 --pretty='%h'`
 export CRI_DOCKERD_LDFLAGS=-ldflags "-s -w -buildid=${REVISION} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.Version=${VERSION} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.PreRelease=${PRERELEASE} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.GitCommit=${REVISION}"
 
+all: cri-dockerd
+
+# always run go build
+cri-dockerd: FORCE
+	GOARCH=$(ARCH) go build -trimpath $(CRI_DOCKERD_LDFLAGS) -o $@
+FORCE:
```

Comment:
```suggestion
.PHONY: cri-dockerd
cri-dockerd:
	GOARCH=$(ARCH) go build -trimpath $(CRI_DOCKERD_LDFLAGS) -o $@
```

Reply:
A `.PHONY` target is always dirty; this has the same effect as you creating a rule for a target that will never resolve and making `cri-dockerd` dependent on that.